### PR TITLE
Refactors landmines

### PR DESCRIPTION
**Pull request by Thunder12345**
Quick refactor to make landmine code more readable. Gonna be using these for an away mission I'm working on, so I decided to clean them up before using them.

### CODE REVIEW

File: code/game/objects/effects/mines.dm [hunk 1]
```diff
@@ -24,48 +23,42 @@
...
 
 /obj/effect/mine/kickmine
 	name = "kick mine"
-	icon_state = "uglymine"
-	minepayload = "triggerkick"
\ No newline at end of file
+
+/obj/effect/mine/kickmine/mineEffect(mob/victim)
+	if(isliving(victim) && victim.client)
+		victim << "<font color='red'><b>You have been kicked FOR NO REISIN!<b></font>"
```
Comment (phil235) [inline code/game/objects/effects/mines.dm]: span class = userdanger please.

File: code/game/objects/effects/mines.dm [hunk 2]
```diff
@@ -24,48 +23,42 @@
...
+
 
 /obj/effect/mine/stun
 	name = "stun mine"
-	icon_state = "uglymine"
-	minepayload = "triggerstun"
+
+/obj/effect/mine/stun/mineEffect(mob/victim)
+	if(isliving(victim))
+		victim.Weaken(8)
```
Comment (Miauw62) [inline code/game/objects/effects/mines.dm]: Fairly sure this is 8 ticks, not 8 seconds.

**Comment by Miauw62** [comment]
you might want to add an "effect" var to these landmines and do victim.Weaken(8 * effect) or something like that for easily making more/less powerful mines.

**Comment by Thunder12345** [comment]
Travis why have you abandoned me

File: code/game/objects/effects/mines.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/game/objects/effects/mines.dm]: actually, human life triggers every 2.1 seconds

so this is 16 seconds.

**Pull request merged**